src/data/services/LmsApiService.ts

```diff
@@ -54,6 +54,18 @@ export interface EnterpriseAdminResponse {
     }>;
   };
 }
+
+export interface EnterpriseAdminResponse {
+  data: {
+    uuid?: string;
+    enterprise_customer_user?: number;
+    last_login?: string | null;
+    completed_tour_flows?: any[];
+    onboarding_tour_dismissed?: boolean;
+    onboarding_tour_completed?: boolean;
+  }
+}
```

Comment:
nit: Can break out duplicate data between `EnterpriseAdminResponse` and `EnterpriseAdminsResponse`
```suggestion
export type EnterpriseAdminPayload = {
    uuid?: string;
    enterprise_customer_user?: number;
    last_login?: string | null;
    completed_tour_flows?: any[];
    onboarding_tour_dismissed?: boolean;
    onboarding_tour_completed?: boolean;
};
export interface EnterpriseAdminResponse {
  data: EnterpriseAdminPayload;
};
```